technology/cassandra/src/main/resources/com/github/mizool/technology/cassandra/mapperProducerGenerator.stg

```diff
@@ -1,5 +1,22 @@
 delimiters "$","$"
+
 mapperProducerGenerator(packageName,className) ::= <<
+/**
```

Comment:
I don't want the boilerplate in _generated_ files, I want it in the _template_ only. please turn it into a StringTemplate comment (`$!` and `!$`). also, move it to the top of the file (unless that breaks things).